### PR TITLE
[WFCORE-2337]: Removal of overlay in a server group fails inside a composite.

### PR DESCRIPTION
**Pull request by ehsavoie**
Fixing the way the resource is removed.

Jira: https://issues.jboss.org/browse/WFCORE-2337

### CODE REVIEW

**Comment by jfdenise** [comment]
@ehsavoie with respect to CLI, all new tests are passing. Thanks!

File: controller/src/main/java/org/jboss/as/controller/CompositeOperationHandler.java [hunk 1]
```diff
@@ -113,6 +113,7 @@ public void handleRollback(OperationContext context, ModelNode operation) {
                     ModelNode stepResponse = responseMap.get(stepName);
                     if (stepResponse.hasDefined(FAILURE_DESCRIPTION)) {
                         failureMsg.get(ControllerLogger.ROOT_LOGGER.compositeOperationFailed(), ControllerLogger.ROOT_LOGGER.operation(stepName)).set(stepResponse.get(FAILURE_DESCRIPTION));
+                        failureMsg.get(ControllerLogger.ROOT_LOGGER.compositeOperationFailed(), "Operation: " + ControllerLogger.ROOT_LOGGER.operation(stepName)).set(operationMap.get(stepName));
```
Comment (bstansberry) [inline controller/src/main/java/org/jboss/as/controller/CompositeOperationHandler.java]: Is this change necessary to resolve WFCORE-2337? If not I'd prefer to deal with it separately. We'd need i18n of "Operation:", and I'm not sure whether the output here is right.

Comment (bstansberry) [inline controller/src/main/java/org/jboss/as/controller/CompositeOperationHandler.java]: Not right; this drops the failure details from the top level failure description and replaces them with the full details of the step operation, including params, which I don't really want to output. The format of the address (dmr syntax) isn't great either. If we output operation details it needs to be in addition to the failure info we already report.

Comment (ehsavoie) [inline controller/src/main/java/org/jboss/as/controller/CompositeOperationHandler.java]: Yes, sorry this was a leftover when tracking which remove op was failing
